### PR TITLE
Update showcase.json

### PR DESCRIPTION
**Pull request by niznet89**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

**Comment by sa-github-api** [comment]
Dear @niznet89,

In order to potentially merge your code in this open-source repository and therefore proceed with your contribution, we need to have your approval on DFINITY's [CLA](https://github.com/dfinity/cla/blob/main/CLA.md).

If you decide to agree with it, please visit this [issue](https://github.com/dfinity/cla/issues/378) and read the instructions there. Once you have signed it, re-trigger the workflow on this PR to see if your code can be merged.

— The DFINITY Foundation

File: showcase.json [hunk 1]
```diff
@@ -2853,5 +2853,25 @@
...
+  {
+  "name": "Tali AI ",
+  "description": "Tali is a Discord Support bot that answers common developer queries.",
+  "website": "https://trytali.com ",
+  "logo": "/img/showcase/tali-ai_logo.webp",
+  "video": "/img/showcase/tali-ai_video.mp4",
+  "display": "Normal",
+  "id": "tali-ai",
+  "oneLiner": "Tali is a Discord Support bot that answers common developer queries.",
+  "stats": "100k+ devs spt. ",
```
Comment (jennifertrin) [inline showcase.json:2866]: Can you please advise what does `spt.` mean?

File: showcase.json [hunk 2]
```diff
@@ -2853,5 +2853,25 @@
   "twitter": "https://twitter.com/221BravoApp",
   "youtube": "",
   "submittableId": "36556639"
+},
+  {
+  "name": "Tali AI ",
```
Comment (jennifertrin) [inline showcase.json:2858]: Can you please remove the extra space after AI?

**Review by jennifertrin — CHANGES_REQUESTED** [review]
Can you please review the comments and also go through the CLA procedure?

File: showcase.json (hunk 1 above)
Comment (niznet89) [reply showcase.json:2866]: Hey Jennifer - spt. is supposed to be 'supported'. I was working with the character constraints. 

If I can manually change it in the JSON, happy to do so.

File: showcase.json (hunk 2 above)
Comment (niznet89) [reply showcase.json]: This is done.

File: showcase.json (hunk 1 above)
Comment (jennifertrin) [reply showcase.json]: Thank you for the clarification. Would you be open to just mentioning `100k+ devs`?

Comment (niznet89) [reply showcase.json]: Yeah that's fine!

Comment (jennifertrin) [reply showcase.json]: Thank you for updates and responses! :)

**Review by jennifertrin — APPROVED** [review]

**Pull request merged**